admin_guide/sdn_troubleshooting.adoc

```diff
@@ -256,75 +256,86 @@ If all of them worked, then jump to the <<debugging-kubernetes>>
 section to work out why the service IP address isn't working.
 
 
+[[debuggin-node-to-node-networking]]
 == Debugging Node to Node Networking
 
 Using our list of non-working endpoints, we need to test connectivity
-to the node.  Let's start by making sure that all of our nodes have
-the expected IP addresses:
+to the node. 
+
+. Make sure that all nodes have the expected IP addresses:
++
+====
 ....
 # oc get hostsubnet
 NAME                   HOST                   HOST IP           SUBNET
 rh71-os1.example.com   rh71-os1.example.com   192.168.122.46    10.1.1.0/24
 rh71-os2.example.com   rh71-os2.example.com   192.168.122.18    10.1.2.0/24
 rh71-os3.example.com   rh71-os3.example.com   192.168.122.202   10.1.0.0/24
 ....
+====
++
+If you are using DHCP they could have changed. Ensure the hostnames, IP
+addresses, and subnets match what you expect. If any node details have changed,
+use `oc edit hostsubnet` to correct the entries.
 
-Look over that list and make sure the hostnames, IP addresses, and subnets match what you expect.  Especially if you are on DHCP and they could have changed.  If that is the problem, use `oc edit hostsubnet` to open an editor, and then correct the entries.
-
-Next, now that we have ensured the node addresses and hostnames are correct, get a list of the endpoint IPs and node IPs:
+. After ensuring the node addresses and hostnames are correct, list the endpoint
+IPs and node IPs:
++
+====
 ....
-# oc get pods --selector=docker-registry --template='{{range .items}}HostIP: {{.status.hostIP}}   PodIP: {{.status.podIP}}{{end}}{{"\n"}}'  # Again, replace docker registry with the one you are interested in
-HostIP: 192.168.122.202   PodIP: 10.128.0.4
+# oc get pods --selector=docker-registry --template='{{range .items}}HostIP: {{.status.hostIP}}   PodIP: {{.status.podIP}}{{end}}{{"\n"}}' HostIP: 192.168.122.202   PodIP: 10.128.0.4
```

Comment:
This could use a line break in the command itself, but it also looks like the output has now gotten merged into the command. I believe it should look more like this:

````
# oc get pods --selector=docker-registry \
    --template='{{range .items}}HostIP: {{.status.hostIP}}   PodIP: {{.status.podIP}}{{end}}{{"\n"}}'

HostIP: 192.168.122.202   PodIP: 10.128.0.4
````